apps/www/components/Questionnaire/Submissions/PersonViewHeaderShare.js

```diff
@@ -0,0 +1,61 @@
+import { useState } from 'react'
+
+import { IconButton } from '@project-r/styleguide'
+import { trackEvent } from '@app/lib/analytics/event-tracking'
+
+import { postMessage, useInNativeApp } from '../../../lib/withInNativeApp'
+import { useTranslation } from '../../../lib/withT'
+
+import { IconShare } from '@republik/icons'
+
+import ShareOverlay from '../../ActionBar/ShareOverlay'
+
+export const HeaderShare = ({ meta, noLabel }) => {
+  const { t } = useTranslation()
+  const { inNativeApp } = useInNativeApp()
+  const [overlay, showOverlay] = useState(false)
+  const { url, title } = meta
+
+  return (
+    <>
+      <IconButton
+        label={!noLabel && t('article/actionbar/share')}
+        labelShort=''
+        Icon={IconShare}
+        href={url}
+        onClick={(e) => {
+          e.preventDefault()
+          trackEvent(['ActionBar', 'shareQuestionnaireLink', url])
```

Comment:
Need to check myself, but does this still work with Plausible? Probably better to use the useTrackEvent hook